File: Prak 5/C/PRAK404-2210817110005-MIZZUDDINABDISSALAM.c
#include <stdio.h>

int main(){
    int p;
    int npp, nkk;
    float np, nk, hl;

    while(p != 5){
        printf("Pilih program \n1. Penjumlahan \n2. Pengurangan \n3. Perkalian \n4. Pembagian \n5. Exit\n");
        printf("Masukkan Pilihan : "); scanf("%d", &p);
        if(p == 5){printf("Terimakasih, telah menggunakan kalkulator M Izzuddin Abdis Salam\n\n");}
        else if(p < 0 || p > 5){printf("Input anda salah, silahkan coba lagi.\n\n");}
        else{
            printf("Masukkan nilai pertama : "); scanf("%f", &np);
            printf("Masukkan nilai kedua   : "); scanf("%f", &nk);
            if(p == 1){
            hl = np + nk;
            printf("Hasil Penjumlalahan antara %.2f dengan %.2f adalah %.2f\n\n", np, nk, hl);}
            else if(p == 2){
            hl = np - nk;
            printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", np , nk, hl);}
            else if(p == 3){
            hl = np * nk;
            printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n\n", np , nk, hl);}
            else {
            hl = np / nk;
            printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n\n", np , nk, hl);}
        }
    }
return 0;}

File: Prak 5/Python/PRAK404-2210817110005-MIZZUDDINABDISSALAM.py
a = 1
while a != 5 :
        print("Pilih program \n1. Penjumlahan \n2. Pengurangan \n3. Perkalian \n4. Pembagian \n5. Exit\n");
        p = float(input("Masukkan Pilihan : "))
        if p == 5 :
            print("Terimakasih, telah menggunakan kalkulator M Izzuddin Abdis Salam\n\n")
            break
        elif p < 0 or p > 5 : print("Input anda salah, silahkan coba lagi.\n\n")
        else : 
            np = float(input("Masukkan nilai pertama : "))
            nk = float(input("Masukkan nilai kedua   : "))
            if p == 1 : 
                hl = np + nk 
                print("Hasil Penjumlalahan antara %.2f dengan %.2f adalah %.2f\n\n"%(np, nk, hl))
            elif p == 2 : 
                hl = np - nk 
                print("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n\n"%(np, nk, hl))
            elif p == 3 : 
                hl = np * nk 
                print("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n\n"%(np, nk, hl))
            else : 
                hl = np / nk 
                print("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n\n"%(np, nk, hl))